apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx

```diff
@@ -171,7 +176,9 @@ const ApplicationReview = ({
               {intl.formatMessage({
                 defaultMessage:
                   "When you submit your application, a copy of your profile will be created as a part of your application.",
-                id: "WWoPd/",
+                id: "G4nPxX",
+                description:
+                  "List item message for the application review page.",
```

Comment:
Consider changing the three descriptions to something unique:

- List item note 1 for the application review page.
- List item note 2 for the application review page.
- List item note 3 for the application review page.

Reply:
Fixed by [a972a4e](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6600/commits/a972a4ed7e18c3b9fb1392ec7b859fbc5815cc4c)